pkg/tanka/parallel.go

```diff
@@ -92,6 +80,7 @@ type parallelOut struct {
 
 func parallelWorker(jobsCh <-chan parallelJob, outCh chan parallelOut) {
 	for job := range jobsCh {
+		log.Printf("Loading %s from %s", job.opts.Name, job.path)
```

Comment:
Supposed to be here? Sounds quite verbose imo

Reply:
I left it intentionally. It gives a bit of feedback to the user as opposed to no feedback right now. I'm happy to remove or accept other suggestions to give a sense of progress.

Reply:
Fair enough. I'm not totally familiar, does this codepath affect any other actions than export?

Reply:
Not that I know off.